lighthouse-core/audits/bootup-time.js

```diff
@@ -34,6 +34,14 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
+// These trace events, when not triggered by a script inside a particular task, are just general Chrome overhead.
+const BROWSER_TASK_NAMES_SET = new Set([
+  'CpuProfiler::StartProfiling',
```

Comment:
I feel this `StartProfling` might be good in a category by itself. It's not typical Chrome tasks, it's the overhead of profiling the page.

Reply:
yeah this felt like a dicey situation so just calling it `Chrome` felt like a decentish balance, really want `Other` to mean `Unknown` maybe we just rename it?

Reply:
Yes

Reply:
after discussion... `Browser` and `Browser GC`